README.md

```diff
@@ -30,19 +33,30 @@ You can find the changes for each version in the [change log](https://github.com
 compile group: 'com.microsoft.azure', name: 'msal4j', version: '1.0.0'
 ```
 
+## Usage
+
+MSAL4J supports multiple [application types and authentication scenarios](https://docs.microsoft.com/en-us/azure/active-directory/develop/authentication-flows-app-scenarios).

```

Comment:
please remove en-us (so that customers get a localized version of the docs)

Reply:
Updated